src/components/Register/Register.js

```diff
@@ -16,46 +25,56 @@ function Signup({ isOpen, onClose, onSubmit, onSigninClick }) {
                     aria-label="close"
                     className="popup__close-button-signup"
```

Comment:
`className="popup__close-button-signup"`


Every close icon should have a common class `popup__close-button`:

The idea why all common elements should have **common** css classes is that you can create common `css` styles, common **universal** functions, find all common elements in the DOM and so on…

Also there should be common classes for `cross` button, `form`, `input`, `submit` button and `overlay`

Reply:
Done